src/PrestaShopBundle/Resources/views/Admin/Improve/Design/MailTheme/Blocks/configuration_form.html.twig

```diff
@@ -22,6 +22,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
```

Comment:
Is this for making an implicit import explicit?

Reply:
This is required starting from Twig 2, as macro functions are not shared across children templates. According to the @twigphp/Twig core team, this was a side effect of Twig 1.x and never intended to be used this way.

Sadly [we made the related test suite optional](https://github.com/PrestaShop/PrestaShop/blob/develop/.travis.yml#L64) so I had to fix it in this pull request when before we never had to think about it: technical debt have to be paid.

Better now than the day of no one can figure out why it's broken :sunglasses: